styleguide/source/assets/scss/03-organisms/_category-navigation-menu.scss

```diff
@@ -88,6 +91,13 @@
       @extend %ama__type--14px;
       padding: $gutter/4 $gutter;
     }
+
```

Comment:
the divider class was added to prevent us from having to loop through an array to simply put a UL inside of a UL element